### PR TITLE
tutorial of distributed training on Yarn

### PR DESCRIPTION
**Pull request by seusofthd**

### CODE REVIEW

**Comment by winstywang** [comment]
Update then I will merge in

**Comment by seusofthd** [comment]
Great, thanks

**Comment by tqchen** [comment]
@seusofthd can you update the PR with the latest dmlc-core code? Then we can merge it in

**Comment by seusofthd** [comment]
I just sent a PR. Tianqi wants me to merge my previous commits but it fails
when I try to follow xgboost's tutorial on conflict merge. Finally solve it
by forking again the project. Is it OK?

**Comment by tqchen** [comment]
yes! Thanks for the effort and patience on this

**Comment by seusofthd** [comment]
Thank you. Next time I will change another editor to write the code for
merge since vim tab causes so much trouble...

**Comment by tqchen** [comment]
@seusofthd Can you followup on this issue?

**Comment by seusofthd** [comment]
@tqchen Sorry, I just come back from spring break, what else do I need to update except my previous PR request?

File: doc/tutorial/yarn_how_to.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+```
+./dmlc-yarn.py -hip 192.168.1.1 -n 1 -s 1 --yarn-scripts xxx/mxnet/dmlc-core/yarn/ python imagenet.py
+```
+In this command, -hip specifies the ip address of resourcemanager, -n specifies the number of workers, -s specifies the number of servers, --yarn-scripts specifies the folder for dmlc-yarn.jar.
+python imagenet.py is the command to execute the imagenet.py.
+
+There is a tricky point here. Let's suppose the number of workers is n and the number of servers is m. The number of containers available in the cluster should be no less than m+n+1 because the ApplicationMaster also need one container to run.  
+
+## Fault Tolerance
+In current ApplicationMaster, whole application will be killed if some of containers fails. You can add the fault tolerance feature by adding a blacklist in ApplicationMaster. Each time one container fails, the node should be added to the blacklist and never be used for execution.
```
Comment (wallyell) [inline doc/tutorial/yarn_how_to.md]: Can you offer me this version of mxnet?

**Pull request closed without merging**